verify-migrations.sh

```diff
@@ -1,7 +1,2 @@
 #! /bin/bash
-
-set -e
-
-if python manage.py makemigrations --dry-run --exit; then
-    exit 1
-fi
+exec python manage.py makemigrations --check
```

Comment:
Do we need use --dry-run --check? Seems --check is same as --exit?

Reply:
Seems that `--check` doesn't change DB either (same as `--dry-run`).

Reply:
Sorry, of course the commands don't change DB, but both the new command and the old one doesn't create any migration files.